app/src/main/java/com/ciandt/dragonfly/example/features/feedback/FeedbackActivity.kt

```diff
@@ -282,9 +282,11 @@ class FeedbackActivity : BaseActivity(), FeedbackContract.View {
 
     private fun showOtherClassifications(otherClassifications: List<Classifier.Classification>) {
         if (otherClassifications.isEmpty()) {
-            footer.visibility = View.GONE
+            otherPredictionsSeparator.visibility = View.GONE
```

Comment:
Yes, this is what will happen now after this commit.

Hide/Show was in footer view. Now, footer is always visible.
The if here is to hide/show the chips view when there are no other classifications.

After this commit:
![screenshot_20170823-143407](https://user-images.githubusercontent.com/68348/29578811-f5657b54-8746-11e7-93d9-38a03849490b.png)